assets/server.ts

```diff
@@ -80,12 +82,17 @@ class Server {
      * @param {Object} data
      * @return {Promise}
      */
-    post(url: any, data: any, etag?: string, rawResponse = false) {
+    post(
+        url: any, 
+        data: any, 
+        etag?: string, 
+        config: { parseJson?: boolean } = {parseJson: true}
```

Comment:
I would call it `options` in general, we use `config` for the client/server config

Reply:
and seems like you could define an interface and use it for both post and checkStatus

Reply:
as we already use options here as a function https://github.com/superdesk/newsroom-core/pull/1214/files/5171c7af00ee55da154aedd46bd58b0b3a996361..a24ad9e7e4d1cc02e91057fb1877d056e761a086#diff-2fb41b398d7dae3e0e795294a461513a587ea451bd7f36ab6887f81978848e10R91.
that's why I use diff name

Reply:
I see, ok np